src/agent/communicator/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.12.4)
+
+project(Communicator)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_BUILD_TYPE Release)
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g")
+
+# Enable testing
+enable_testing()
+
+include_directories(include)
+include_directories(${CMAKE_SOURCE_DIR}/communicator)
```

Comment:
From reading the documentation it looks like line 12 already adds the header files for this folder, making line 13 redundant. See https://cmake.org/cmake/help/v3.22/command/include_directories.html

Reply:
Fixed